packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -395,9 +403,25 @@ const ChartTrade = observer(props => {
             yAxisMargin={{
                 top: isMobile() ? 76 : 106,
             }}
+            isLive={true}
+            leftMargin={isDesktop() && is_positions_drawer_on ? 328 : 80}
+            is_beta={is_beta_chart}
         >
-            <ChartMarkers />
-            {is_accumulator && (
+            {!is_beta_chart && <ChartMarkers />}
+            {is_accumulator && is_beta_chart && (
+                <AccumulatorsChartElementsBeta
+                    all_positions={all_positions}
+                    current_spot={current_spot}
+                    current_spot_time={current_spot_time}
+                    has_crossed_accu_barriers={has_crossed_accu_barriers}
+                    should_show_profit_text={
+                        !!accumulator_contract_barriers_data.accumulators_high_barrier &&
+                        getDecimalPlaces(currency) <= 2
+                    }
+                    symbol={symbol}
+                />
+            )}
+            {is_accumulator && !is_beta_chart && (
```

Comment:
We can use the same component. I don't think there would be any problem. Did it throw any error?